packages/sdk/package.json

```diff
@@ -0,0 +1,70 @@
+{
+	"name": "@directus/sdk",
+	"version": "9.0.0-rc.49",
+	"description": "The official Directus SDK for use in JavaScript!",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/directus/directus.git"
+	},
+	"main": "./dist/index.js",
+	"module": "./dist/sdk.esm.js",
```

Comment:
we're defining a module entrypoint, and defining `axios` as a `dependency`, but based on your rollup output we are also pre-bundling `axios`... 

if this is intended, then it's better to move `axios` to a `devDependency` as it's already being included part of your dist output and there is no need for it on another dev's machine

normally you would leave this to the end-consumer and let their bundle handle it.

the only time you _should_ bundle is if you're shipping directly to browsers. this might be the case, but at that point i would make that a separate build.



with the introduction of `ITransport` there may be some value in looking to make `fetch` your default transport. this would ultimately remove this dependency and make your package even leaner.

Reply:
Yeah, I mentioned using fetch to Rijk a while ago, but I haven't done anything with it yet, so I'll have to take a look at it first, but it makes sense to do this instead. Though I think `axios` would land on `{peer/optional}Dependencies` in this case right?

Reply:
you know what, i just checked out the `cjs` output... it looks like the `require` is intact there. so we still need `axios` listed as a dependency, otherwise using this in a node server context will fail.

no issues leaving it. the only thing up for debate really is whether we should be pre-bundling... i don't think it makes to do it.

if I for example have an existing react-app and i use `axios` for my API, and then need to integrate with `directus` i would expect to see two versions of `axios` now. one bundled in my app, and one pre-bundled via directus. 

if we do not pre-bundle, then we eliminate this duplication and offload it to the consumer of the module.

Reply:
I think bundle/not bundle stuff really depends on what you'll be importing on your project. 

IMHO these outputs shouldn't carry dependencies, so I`ll need to triple check the output. 

Additional outputs can be added with a `.bundle` suffix (as I think we had before). I haven`t added them because I was unsure how it should look like and/or behave.

Reply:
> IMHO these outputs shouldn't carry dependencies, so I`ll need to triple check the output.

Agreed. Dependencies should be external _except_ for the single minified "plop it in a `<script>` and you're done"  `.min.js` file